content/attacks/posts/2023-11-23-HecoBridge.md

```diff
@@ -20,9 +20,9 @@ On November 22, 2023, Heco Bridge and HTX Exchange were victims of cyberattacks,
 
 ## Attackers
 
-The attacker has yet to be identified but used the following wallets to transfer some of the high-jacked funds:
+The attacker has yet to be identified but used the following wallets to transfer some of the hijacked funds:
 
-        - 0xe47e6dA16Bb83EB0FD26b3F29b15CE8Fab089B9e (holding wallet)
+        - 0xe47e6dA16Bb83EB0FD26b3F29b15CE8Fab089B9e
```

Comment:
Please put a link to Etherscan for the address

Reply:
Link added.